src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Errors/PossibleErrors.cs

```diff
@@ -61,5 +61,14 @@ public static class PossibleErrors
             .Add(ErrorId.AccountAdmin_TypeOfRole_MissingSelection, "Select the type of user you are adding")
             .Add(ErrorId.AccountAdmin_TypeOfUserLa_MissingSelection, "Select what they need to do")
             .Add(ErrorId.AccountAdmin_TypeOfUserVcs_MissingSelection, "Select what they need to do")
+            .Add(ErrorId.AccountAdmin_UserEmail, "Enter an email address")
```

Comment:
Maybe the key could indicate what the error is. For example:

ErrorId.AccountAdmin_UserName -> ErrorId.AccountAdmin_UserName_Missing

etc